bugsnag-android-core/src/main/java/com/bugsnag/android/SessionFilenameInfo.kt

```diff
@@ -27,29 +29,62 @@ internal data class SessionFilenameInfo(
          * Generates a filename for the session in the format
          * "[UUID][timestamp]_v2.json"
          */
-        fun toFilename(timestamp: Long, uuid: String): String {
-            return "${uuid}${timestamp}_v2.json"
+        fun toFilename(apiKey: String, timestamp: Long, uuid: String): String {
+            return "${apiKey}_${uuid}${timestamp}_v2.json"
```

Comment:
```suggestion
            return "${apiKey}_${uuid}${timestamp}_v3.json"
```

Adding "v3" to the end allows us to more easily detect the differences between the existing Session files and the new format (which includes the apiKey in the filename).